snapcraft/internal/lxd.py

```diff
@@ -111,7 +116,7 @@ def execute(self, step='snap', args=None):
             self._setup_project()
             self._wait_for_network()
             self._container_run(['apt-get', 'update'])
```

Comment:
side comment, we need a better plan for this as it is really too slow, maybe piggy back it to `snapcraft update` and if it is a containerized instance we do `update` for the parts cache, apt cache and a snap refresh.